saleor/graphql/core/mutations.py

```diff
@@ -196,6 +196,12 @@ def get_nodes_or_error(cls, ids, field, only_type=None, qs=None):
             )
         return instances
 
+    @classmethod
+    def get_duplicates_ids(cls, add_nodes, remove_nodes):
```

Comment:
I see two small things to improve here:
- it's rather a utility function, it just checks two lists for common elements so I think it should be moved to some utils module, e.g. `graphql/core/utils`
- let's add a docstring e.g. "Return items that appear on both provided lists."